helm/prometheus-rules/templates/alerting-rules/job.rules.yml

```diff
@@ -14,14 +14,27 @@ spec:
     - alert: ManagementClusterJobFailed
       annotations:
         description: '{{`Job {{ $labels.namespace }}/{{ $labels.job_name }} is failed.`}}'
-      expr: kube_job_failed{cluster_type="management_cluster", condition="true"} == 1
+      expr: kube_job_failed{cluster_type="management_cluster", condition="true", job_name!~"silence-operator-sync.+|grafana-permission.+"} == 1
       for: 15m
       labels:
         area: kaas
         cancel_if_outside_working_hours: {{ include "workingHoursOnly" . }}
         severity: notify
         team: {{ include "providerTeam" . }}
         topic: managementcluster
+    - alert: AtlasJobHasNotBeenScheduledForTooLong
+      annotations:
+        description: '{{`CronJob {{ $labels.namespace }}/{{ $labels.cronjob }} has not been scheduled for more than 1 day.`}}'
+        opsrecipe: job-has-not-been-scheduled-for-too-long/
+      # This alert triggers when the silence sync or grafana permission job did not schedule for more than 1 day
+      # or if the job did not run successfully at least once in the last day
+      expr: (time() - kube_cronjob_status_last_schedule_time{cronjob=~"silence-operator-sync|grafana-permissions"} ) > 86400
+            or count(max_over_time(kube_job_status_succeeded{job_name=~"silence-operator-sync.+|grafana-permission.+"}[1d]) == 1) == 0
```

Comment:
I would rather have 2 different alert for `silence-operator-sync` and `grafana-permission` job.

Reply:
alerts are splitted